code review

swdanielli opened the pull request:
Hi @pmitros @singingwolfboy ,

This is a module we previously built to allow students to recommend useful resources for problems and share resources with their fellows. We have tested it in two runs of 6.00.1x. We are now attempting to make it into production. I was wondering could you help us do the code review.

Comparison: https://github.com/pmitros/RecommenderXBlock/compare/b4f3596d095d88bb4540f0d04ab81836d84edc98...c932155364d31db354c6d289464c02800aeda8d1

Besides, there are testcases for this module: https://github.com/edx/edx-platform/pull/4845

pmitros commented:
@singingwolfboy Done: https://github.com/pmitros/RecommenderXBlock/pull/26

singingwolfboy commented:
@swdanielli @pmitros Do you still want to update the RecommenderXBlock in edx-platform? If so, you should rebase this pull request, and make sure that it references the correct commit you want in the RecommenderXBlock repository.

swdanielli commented:
This is the recommender we reviewed. Mark gave a thumbs-up for the accessibility. Incorporated feedback from Piotr and and DB. Squashed, rebased, and updated test cases. @singingwolfboy @pmitros, mind giving a quick :+1:, and I'll merge?

singingwolfboy commented:
:+1: after tests pass on Jenkins

pmitros commented:
:+1: once tests pass.
